litho-widget/src/main/java/com/facebook/litho/widget/ComponentRenderInfo.java

```diff
@@ -76,6 +76,11 @@ public Builder component(Component component) {
       return this;
     }
 
+    public Builder component(Component.Builder builder) {
+      this.component = builder.build();
```

Comment:
I think a more common pattern here would just to do `return component(builder.build());`, what do you think? 

Thanks for doing this!

Reply:
@IanChilds sure that would save possible issues in case more things are added to `.component(Component component)` method! Will change it!